companies/tasks/kis_market_amount.py

```
@@ -0,0 +1,107 @@
+
+    각 기업별로 별도 Celery task를 생성하여 병렬 처리
+    (단, Celery worker 수에 따라 실제 병렬도 결정)
+    """
+    companies = Company.objects.filter(is_deleted=False)
+    total = companies.count()
+    logger.info(f"전체 기업 시가총액 갱신 작업 등록 시작: {total}개 기업")
+
+    for company in companies:
+        try:
+            sync_market_amount.delay(company.stock_code)
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)